packages/components/src/common/markdownNodeTypes.js

```diff
@@ -0,0 +1,18 @@
+export const MARKDOWN_NODE_TYPES = {
```

Comment:
Could we use the enum style here as well?
```suggestion
export const MarkdownNodeType = {
```